.mk/development.mk

```diff
@@ -0,0 +1,66 @@
+##@ Development
+
+.PHONY: create-kind-cluster
+create-kind-cluster: $(KIND) ## Create kind cluster with OVN-k8s
+	ln -sf $$GOPATH/bin/kind-v0.11.1 $$GOPATH/bin/kind; \
+	sudo yum -y install python-pip; \
+	mkdir -p $$GOPATH/src/github.com/ovn-org; \
+	cd $$GOPATH/src/github.com/ovn-org; \
+	git clone git@github.com:ovn-org/ovn-kubernetes.git || true; \
+	cd ovn-kubernetes/contrib; \
+	./kind.sh; \
```

Comment:
This fixes the issue I have:
`export KIND_IMAGE=docker.io/kindest/node && ./kind.sh`

Reply:
removed that area